p5/core/constants.py

```diff
@@ -44,6 +44,7 @@
 RIGHT = "RIGHT"
 TOP = "TOP"
 BOTTOM = "BOTTOM"
+BASELINE = "BASELINE"
```

Comment:
I think it's beneficial to document what these constants are used for, like the other constants in this file.

Reply:
Updated